lang/en/logstore_xapi.php

```diff
@@ -89,6 +89,8 @@
 $string['logstorexapierrorlog'] = 'Logstore xAPI Error Log';
 $string['noerrorsfound'] = 'No errors found';
 $string['logstorexapi:viewerrorlog'] = 'View xAPI error log';
+$string['failed_transform_errortype'] = '101';
```

Comment:
101?

Reply:
In the absence of an official error code I uses 101 as a distinct error code.

Reply:
I think we want to remove the underscores here, you can keep underscores below for privacy:metadata:logstore_xapi_log

Reply:
I have fixed this.